api/graphql/mutations/sequencing_groups.py

```diff
@@ -0,0 +1,32 @@
+import strawberry
+
+from db.python.layers.sequencing_group import SequencingGroupLayer
+from models.models.sequencing_group import SequencingGroupUpsertInternal
+from models.utils.sequencing_group_id_format import (  # Sample,
+    sequencing_group_id_transform_to_raw,
+)
+
+
+@strawberry.type
+class SequencingGroupsMutations:
+    """Sequencing group mutations"""
+
+    @strawberry.mutation
+    async def update_sequencing_group(
+        self,
+        sequencing_group_id: str,
+        sequencing_group: strawberry.scalars.JSON,
+        info: strawberry.types.Info,
+    ) -> bool:
+        """Update the meta fields of a sequencing group"""
+        connection = info.context['connection']
```

Comment:
We'd need to make sure the connection it gets here is a `write` connections, or a projectless connection, and then decide auth later on. Otherwise it won't allow writes correctly